session.go

```diff
@@ -186,6 +191,10 @@ func Sessioner(options ...Options) macaron.Handler {
 
 		ctx.Next()
 
+		if ctx.Req.Header.Get("Upgrade") == "websocket" && manager.opt.IgnoreReleaseForWebSocket {
```

Comment:
`manager.opt.IgnoreReleaseForWebSocket` should come before `ctx.Req.Header.Get("Upgrade") == "websocket"`.